src/components/space/trajectory.tsx

```diff
@@ -14,7 +14,7 @@ export type TrajectoryProps = {
   color: Color;
   endDate: number;
   tle: string;
-  visible?: boolean;
+  visible: boolean;
```

Comment:
This `visible` prop could actually stay optional to be consistent with React Three Fiber's types.

![image](https://user-images.githubusercontent.com/82393815/208163948-01cf7883-38cb-4801-ac2e-cae2216823ae.png)